.travis.yml

```diff
@@ -22,10 +22,6 @@ install:
 script:
   - ./vendor/bin/phpunit
 
-jobs:
```

Comment:
The `jobs` line is still necessary as the resulting YAML file will be invalid otherwise.

Reply:
Ok, i'll get it back